### ISSUE TITLE
[1.0.14] Sony xPeria Z / Android 5.1.1 - Text over-write Error - During turn off wifi and switch back to previous screen

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-android-issues#48]
Frequency: Every time

Action:
1. Turn OFF Internet
2. Launch Lomotif
3. Select Start
4. Tap on search bar for motifs
5. It will give you error message "An error has occured"
6. Click on Cancel button right to Search bar
7. Observe the screen

Expected result: Clicking on cancel button should redirect to previous My Video screen with no error 

Actual result: My Video screen appears but with previous error message. Previous error message is over-write on My Video screen

![screenshot_2016-04-02-10-40-10](https://cloud.githubusercontent.com/assets/9691390/14224698/71e80b2e-f8c3-11e5-9fc7-0cb9c4618425.png)

**Comment by xiuie** [comment]
@hiteshbheda Does this affect the functionality of the app?

**Comment by hiteshbheda** [comment]
@xiuie @jpaulo1205 No, this does not affect to functionality but overlapping of previous error message with folder name confuse user to select First folder.

**Comment by jpaulo1205** [comment]
Note: Fixed in 1.0.16

**Comment by vysohanych** [comment]
I can't reproduce it on build 1.0.18 on Asus Nexus 7 Android 5.1.1

**Comment by hiteshbheda** [comment]
@xiuie @jpaulo1205 I can't reproduce it on build [1.0.19.1] on Asus Zenfone Go 4.5 / Android 5.1
